validator/db/kv/attestation_history_new.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/pkg/errors"
```

Comment:
Shall we rename these to v2 to be more semantic?

Reply:
renamed, looks much better